mollie/payments.go

```diff
@@ -43,44 +43,60 @@ const (
 	Amount                          *Amount         `json:"amount,omitempty"`
 	AmountRefunded                  *Amount         `json:"amountRefunded,omitempty"`
 	AmountRemaining                 *Amount         `json:"amountRemaining,omitempty"`
 	AmountCaptured                  *Amount         `json:"amountCaptured,omitempty"`
-	Description                     string          `json:"description,omitempty"`
-	RedirectURL                     string          `json:"redirectUrl,omitempty"`
-	WebhookURL                      string          `json:"webhookUrl,omitempty"`
-	Method                          PaymentMethod   `json:"method,omitempty"`
-	Metadata                        interface{}     `json:"metadata,omitempty"`
-	Locale                          Locale          `json:"locale,omitempty"`
-	CountryCode                     string          `json:"countryCode,omitempty"`
-	ProfileID                       string          `json:"profileId,omitempty"`
 	SettlementAmount                *Amount         `json:"settlementAmount,omitempty"`
-	SettlementID                    string          `json:"settlementId,omitempty"`
-	CustomerID                      string          `json:"customerId,omitempty"`
-	SequenceType                    SequenceType    `json:"sequenceType,omitempty"`
-	MandateID                       string          `json:"mandateId,omitempty"`
-	OrderID                         string          `json:"orderId,omitempty"`
 	ApplicationFee                  *ApplicationFee `json:"applicationFee,omitempty"`
-	Links                           PaymentLinks    `json:"_links,omitempty"`
 	Details                         *PaymentDetails `json:"details,omitempty"`
+	CreatedAt                       *time.Time      `json:"createdAt,omitempty"`
+	AuthorizedAt                    *time.Time      `json:"authorizedAt,omitempty"`
+	PaidAt                          *time.Time      `json:"paidAt,omitempty"`
+	CanceledAt                      *time.Time      `json:"canceledAt,omitempty"`
+	ExpiresAt                       *time.Time      `json:"expiresAt,omitempty"`
+	ExpiredAt                       *time.Time      `json:"expiredAt,omitempty"`
+	FailedAt                        *time.Time      `json:"failedAt,omitempty"`
+	Mode                            Mode            `json:"mode,omitempty"`
+	Locale                          Locale          `json:"locale,omitempty"`
 	RestrictPaymentMethodsToCountry Locale          `json:"restrictPaymentMethodsToCountry,omitempty"`
-	SubscriptionID                  string          `json:"subscriptionId,omitempty"`
-	TestMode                        bool            `json:"testmode,omitempty"`
+	Method                          PaymentMethod   `json:"method,omitempty"`
+	Links                           PaymentLinks    `json:"_links,omitempty"`
+	SequenceType                    SequenceType    `json:"sequenceType,omitempty"`
+	// Payment method specific params.
+	DigitalGoods         bool                   `json:"digitalGoods,omitempty"`
```

Comment:
There is some inconsistency here with the indentation compared to the original parameters.

Reply:
yeah, it is because of the comment in the middle of the struct declaration, I will fix it and then merge